### PR TITLE
Improve PythonFdWrapper class

### PR DESCRIPTION
**Pull request by arogge**
This PR tries to improve the PythonFdWrapper.
With these changes applied
* exceptions will be caught within the python code and made into JobMessages
* `from PythonFdWrapper import *` will now only import some of the names
* pylint-compliance

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)
- [Boy Scout Rule](https://docs.bareos.org/DeveloperGuide/generaldevel.html#boy-scout-rule)

### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)
Make sure you check/merge the PR using `devtools/pr-tool` to have some simple automated checks run and a proper changelog record added.

##### General
- [x] Is the PR title usable as CHANGELOG entry?
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted
~~Required backport PRs have been created~~

##### Source code quality
- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR

### CODE REVIEW

File: core/src/plugins/filed/python/pyfiles/BareosFdWrapper.py [hunk 1]
```diff
@@ -21,95 +21,157 @@
...
+    @wraps(wrapped_func)
+    def wrap_func(*args, **kwargs):
+        try:
+            return wrapped_func(*args, **kwargs)
+        except Exception as e:  # pylint: disable=broad-exception-caught
+            DebugMessage(20, f"{traceback.format_exc()}\n")
+            JobMessage(M_FATAL, f"{wrapped_func.__name__}() failed: {e}\n")
+            return bRC_Error
+
+    return wrap_func
```
Comment (sebsura) [inline core/src/plugins/filed/python/pyfiles/BareosFdWrapper.py:73]: Do you have an example of a situation where an excepction is not caught by the `PyErrorHandler()` in the core ?

Comment (arogge) [reply core/src/plugins/filed/python/pyfiles/BareosFdWrapper.py:73]: I'm not sure the exception was not caught by `PyErrorHandler()`, but I remember having seen cases where there was no meaningful result.

**Comment by arogge** [comment]
The new exception handling is not wanted/required, but all the other improvements should be merged.

**Comment by arogge** [comment]
I made it so that all functions bubble up their exceptions, even if they didn't do that previously. i.e. we had exception handling in `load_bareos_plugin()` previously, but now this also bubbles up to python-fd.

**Pull request merged**